### ISSUE TITLE
500.19 Error 

### ISSUE THREAD

**Issue by Kw3li** [unosquare/passcore#42]
I followed the instructions carefully and this is the error you get when you pull up the server. Any ideas?

Windows 2012 R2 clean install including all windows updates. 

This configuration section cannot be used at this path. This happens when the section is locked at a parent level. Locking is either by default (overrideModeDefault="Deny"), or set explicitly by a location tag with overrideMode="Deny" or the legacy allowOverride="false".

**Comment by crgbt** [comment]
In the Add Roles and Features Wizard, In addition to Role "Web Server (IIS) ensure you have the Role "Application Server" selected and the Feature "Web Server (IIS) Support" (under Application Server) installed, accepting all suggested defaults. This resolved the issue for me on a fresh 2012 R2 build.

**Comment by mariodivece** [comment]
Please try with the latest version and let us know if this is still a problem. Reopen the issue if necessary.

**Comment by BeastDaBest** [comment]
Hi,
I've installed Passcore on Win Server 2019  Fresh install. For the past 3 hours I've been trying everything and read a lot of different article about error 500.19 with error code : 0x8007000d but still can't figure out how to fix this. I'm sorry it's been very frustrating.

I've installed the latest version of the passcore program and Installed all the application requirements,   but still same error.

![image](https://user-images.githubusercontent.com/6089142/80322086-84e93c00-87f0-11ea-8800-08eebc6a1f48.png)

even If I open any of the items below I get this error: 
![image](https://user-images.githubusercontent.com/6089142/80322108-b3671700-87f0-11ea-9de6-0a2f6f5e5506.png)

I installed Passcore twice same error. 

I installed IIS URL Rewrite and .net core 3.0 windows server hosting bundle 
![image](https://user-images.githubusercontent.com/6089142/80322158-148eea80-87f1-11ea-93c7-719d785cfdc3.png)

the Content of my Web.config looks like this : 
![image](https://user-images.githubusercontent.com/6089142/80322172-2ff9f580-87f1-11ea-9295-7f4705a4ec69.png)

any help/advise would be much appreciated.
